srcCpp03/PerftestTransport.cxx

```diff
@@ -639,6 +658,14 @@ std::string PerftestTransport::printTransportConfigurationSummary()
         stringStream << "\tNic: " << allowInterfaces << "\n";
     }
 
+    stringStream << "\tUse Multicast: "
```

Comment:
```
    stringStream << "\tUse Multicast: "
                 << ((allowsMulticast()) ? "True" : "False");
```

Reply:
What is the change here @softwarejimenez?

Reply:
have `<<` at the same height

Reply:
Done. I was looking to the file in the old C++ implementation, sorry.